assets/js/atomic/blocks/product-elements/summary/edit.tsx

```diff
@@ -21,11 +29,4 @@ const Edit = ( { attributes } ) => {
 	);
 };
 
-export default withProductSelector( {
-	icon: BLOCK_ICON,
-	label: BLOCK_TITLE,
-	description: __(
-		'Choose a product to display its short description.',
-		'woo-gutenberg-products-block'
-	),
-} )( Edit );
```

Comment:
This description is different from `BLOCK_DESCRIPTION`. This indicates clearer the expected action from users when choosing a product to display the summary. So we should keep the original description IMO.

But, looking at the shared config, the `ancestor` config is set to `[ '@woocommerce/all-products', '@woocommerce/single-product' ]`, so users will never see the product selector. Because of that, I think we should remove this HOC, and export the `Edit` component as the default.

Reply:
Thanks for pointing this out, @dinhtungdu. As discussed in p1667802571415029-slack-D048XCQ0BEJ, I'll keep this part fort now and [added a to-do to refactor this part in the future](https://github.com/woocommerce/woocommerce-blocks/pull/7566#issuecomment-1305206712).